sympy/integrals/laplace.py

```diff
@@ -997,6 +1000,41 @@ def _laplace_apply_simple_rules(f, t, s):
     return None
 
 
+def _piecewise_to_heaviside(f, t):
+    """
+    This function converts a Piecewise expression to an expression written
+    with Heaviside. It is not exact, but valid in the context of the Laplace
+    transform.
+    """
+    x = piecewise_exclusive(f)
+    r = 0
```

Comment:
```suggestion
    r = S.Zero
```

It could be better to start from SymPy zero, because I'm not sure if this can bleed through raw python integer if the piecewise is empty.

Reply:
I'll change all occurrences of this.